PairedSingleSampleWf_noqc_nocram_optimized.wdl

```diff
@@ -168,8 +171,9 @@ task SamToFastqAndBwaMemAndMba {
         PROGRAM_GROUP_NAME="bwamem" \
         UNMAPPED_READ_STRATEGY=COPY_TO_TAG \
         ALIGNER_PROPER_PAIR_FLAGS=true \
-        UNMAP_CONTAMINANT_READS=true \
-        ADD_PG_TAG_TO_READS=false
+        UNMAP_CONTAMINANT_READS=true
+        #UNMAP_CONTAMINANT_READS=true \

```

Comment:
Why were these parameters commented out? In latest version of this workflow these tools still uses the parameters [link](https://github.com/gatk-workflows/gatk4-genome-processing-pipeline/blob/d78f3886fde1a7146f0318552cb2a335eac70dde/tasks/Alignment.wdl#L106).

Reply:
These changes were something we inherited.  We can revert and test again.

Reply:
A person from Davids team can better determine whether these parameters are necessary, I just wanted to point out the difference and provide a link to the Broads current pipeline.